pkg/assembler/backends/keyvalue/artifact.go

```diff
@@ -225,6 +227,117 @@ func (c *demoClient) artifactExact(ctx context.Context, artifactSpec *model.Arti
 
 // Query Artifacts
 
+func (c *demoClient) ArtifactsList(ctx context.Context, artifactSpec model.ArtifactSpec, after *string, first *int) (*model.ArtifactConnection, error) {
```

Comment:
qq: would it be beneficial to have this be in a pagination struct? or does appolo/frameworks need it to be in the top level API?

Reply:
@mihaimaruseac answered this in his comment - https://github.com/guacsec/guac/issues/1525#issuecomment-1830157381

Reply:
sgtm - tks